astropy/io/fits/tests/test_image.py

```diff
@@ -168,6 +168,53 @@ def test_open_3(self):
         finally:
             fits.conf.lazy_load_hdus = True
 
+    def test_fortran_array(self):
+        # Test that files are being correctly written+read for "C" and "F" order arrays
+        dims = (3,7)
+        size = np.product(dims)
+        a = np.zeros(dims, order='C')
+        b = np.zeros(dims, order='F')
+        a[:] = np.arange(size).reshape(*dims)
+        b[:] = np.arange(size).reshape(*dims)
```

Comment:
Any reason not to use `b = np.asfortranarray(a)` here?

Reply:
no, it should be good as well with `asfortranarray`.